catalog/MDCCatalog/MDCCatalogCollectionViewCell.swift

```diff
@@ -72,6 +84,15 @@ class MDCCatalogCollectionViewCell: UICollectionViewCell {
     label.text = ""
   }
 
+  func updateTheme() {
+    label.font = AppTheme.globalTheme.typographyScheme.button
```

Comment:
Tip: Let's use `containerScheme` instead of `typographyScheme` and `colorScheme` so we don't have to migrate later.

Reply:
If we update this code to use containerScheme, we will need to implement a fallback color scheme due to the nullable nature of the container scheme's properties. When we move forward with nonnull schemes in the container scheme, such code would also have to be migrated to the new API.

With that in mind, the work required to migrate between either of these implementations is similar so I'd prefer the current pattern that makes use of the convenience API rather than implementing explicit nil handling and fallback behavior.